agent/pkg/k8s/certificates.go

```diff
@@ -11,16 +13,19 @@ import (
 	l5dk8s "github.com/linkerd/linkerd2/pkg/k8s"
 	ldTls "github.com/linkerd/linkerd2/pkg/tls"
 	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 )
 
 const (
 	identityComponentName        = "identity"
 	linkerdNsEnvVarName          = "_l5d_ns"
 	linkerdTrustDomainEnvVarName = "_l5d_trustdomain"
+	trustRootsConfigMapName      = "linkerd-identity-trust-roots"
+	trustRootsConfigMapKeyName   = "ca-bundle.crt"
```

Comment:
are these easily available in the `linkerd/linkerd2` module?

Reply:
Unfortunately, no